test/components/virtual-with-if.riot

```diff
@@ -0,0 +1,9 @@
+<virtual-with-if>
+  <ol>
+    <li>1</li>
+    <virtual if={ props.more }>
```

Comment:
I am looking forward to checking this feature but these tests  should be handled [here](https://github.com/riot/dom-bindings) and not in the main riot repo